src/main/java/org/jenkinsci/plugins/docker/workflow/AbstractEndpointStepExecution.java

```diff
@@ -69,8 +69,7 @@
 
     }
 
-    // TODO use BodyExecutionCallback.TailCall from https://github.com/jenkinsci/workflow-plugin/pull/168
-    private static class Callback extends BodyExecutionCallback {
+    private static class Callback extends BodyExecutionCallback.TailCall {
```

Comment:
Part of a commit which updated the `workflow.version`. Without that update, the `fileCredentials` test would fail by running the expected commands but failing to capture the output, which I think was a race condition affecting very quick `sh` steps fixed in some later version of Pipeline.